hack/mirror-crio.sh

```
@@ -50,4 +50,4 @@ do
     mirror_crio_repo_for_version $i
 done
 
-gsutil rsync -d -r $LOCAL_MIRROR_DIR gs://$BUCKET_DIR
+gcloud storage rsync -d -r $LOCAL_MIRROR_DIR gs://$BUCKET_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)